tests/models/mistral/test_modeling_mistral.py

```diff
@@ -535,15 +535,11 @@ def test_model_7b_logits(self):
             out[0, 0, :30], EXPECTED_SLICE[self.cuda_compute_capability_major_version], atol=1e-4, rtol=1e-4
         )
 
-        del model
-        backend_empty_cache(torch_device)
-        gc.collect()
-
     @slow
     @require_bitsandbytes
     def test_model_7b_generation(self):
         EXPECTED_TEXT_COMPLETION = {
-            7: "My favourite condiment is 100% ketchup. I love it on everything. I'm not a big",
+            7: "My favourite condiment is 100% ketchup. I’m not a fan of mustard, mayo,",
```

Comment:
should update in https://github.com/huggingface/transformers/pull/29905 but forgot